new_images.yml

```diff
@@ -4,4 +4,4 @@ new_images:
     use-case: "processing"
```

Comment:
Does this not have the spark-version change?

Reply:
yeah this is for spark 3.1